src/dymaptic.GeoBlazor.Core/dymaptic.GeoBlazor.Core.csproj

```diff
@@ -22,6 +22,7 @@
         <GeneratePackageOnBuild Condition="$(Configuration) == 'RELEASE'">true</GeneratePackageOnBuild>
         <TargetFrameworks>net7.0;net6.0</TargetFrameworks>
         <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+        <UserSecretsId>364f0380-f3fa-441e-8138-e888b38a26c4</UserSecretsId>
```

Comment:
user secrets definitely don't work in a library project

Reply:
removed.